api/src/db-connection.ts

```diff
@@ -82,13 +82,15 @@ export class Connection {
       SELECT paper.*,
              (
                 SELECT COUNT(*)
-                  FROM citation
-                 WHERE citation.paper_id = paper.s2_id
+                  FROM entity
+                 WHERE entity.paper_id = paper.s2_id
+                   AND type = 'citation'
              ) AS citations,
              (
                 SELECT COUNT(*)
-                  FROM symbol
-                 WHERE symbol.paper_id = paper.s2_id
+                  FROM entity
+                 WHERE entity.paper_id = paper.s2_id
+                   AND type = 'symbol'
```

Comment:
I think these queries should take into account the 'version' table. The numbers may be inflated as a single paper may have many versions with 1000s or 10000s of entities. Specifically there is an `entity.version` column and the version table itself is indexed on paper_id. Easiest thing for now is probably aggregating to max(version.version) to find the latest. Happy to address this as a follow-up.

Reply:
Sure, I can take care of that before merging.

Reply:
I took a stab at it -- but now that I'm reviewing it I wonder if it can even simpler.

I think y'all might be saying that there's only one `version` that I need to use (which is the most recent) for all entity types. So I can use one query to find that, and filter the counts based on that.

Let me know if that's the case I can tweak this to reflect that.

Reply:
Ok, this got another tweak, I think it's right now.

Reply:
Yes, that looks right to me. My comment above wasn't quite right as the "version" column referenced in "entity" is actually "index" in the version table. What you've done here sidesteps the need to know that 👍 

I verified against the public schema that there are entities for papers I expect to have them (i.e., not all are 0).

Reply:
I took _another_ stab at the query, with your help (it's more or less what you shared with me in Slack).

TAL.